test/vanilla/version-tolerant/Expected/AcceptanceTests/MediaTypesVersionTolerant/mediatypesversiontolerant/_operations/_operations.py

```diff
@@ -137,206 +105,22 @@ def build_put_text_and_json_body_request(*, content: str, **kwargs: Any) -> Http
-
-    @overload
-    def analyze_body(self, input: Optional[IO] = None, *, content_type: Optional[str] = None, **kwargs: Any) -> str:
-        """Analyze body, that could be different media types.
-
-        :param input: Input parameter. Default value is None.
-        :type input: IO
-        :keyword content_type: Body Parameter content-type. Content type parameter for binary body.
-         Default value is None.
-        :paramtype content_type: str
-        :return: str
-        :rtype: str
-        :raises ~azure.core.exceptions.HttpResponseError:
-        """
-
+class MediaTypesClientOperationsMixin(MixinABC, abc.ABC):
     @distributed_trace
-    def analyze_body(self, input: Optional[Union[JSON, IO]] = None, **kwargs: Any) -> str:
-        """Analyze body, that could be different media types.
-
-        :param input: Input parameter. Is either a model type or a IO type. Default value is None.
-        :type input: JSON or IO
-        :keyword content_type: Body Parameter content-type. Known values are: 'application/json',
-         'application/pdf', 'image/jpeg', 'image/png', 'image/tiff'. Default value is None.
-        :paramtype content_type: str
-        :return: str
-        :rtype: str
-        :raises ~azure.core.exceptions.HttpResponseError:
-        """
-        error_map = {401: ClientAuthenticationError, 404: ResourceNotFoundError, 409: ResourceExistsError}
-        error_map.update(kwargs.pop("error_map", {}) or {})
-
-        _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
-        _params = kwargs.pop("params", {}) or {}
-
-        content_type = kwargs.pop("content_type", _headers.pop("Content-Type", None))  # type: Optional[str]
-        cls = kwargs.pop("cls", None)  # type: ClsType[str]
-
-        _json = None
-        _content = None
-        if isinstance(input, (IO, bytes)):
-            _content = input
-        else:
-            if input is not None:
-                _json = input
-            else:
-                _json = None
-            content_type = content_type or "application/json"
-
-        request = build_analyze_body_request(
-            content_type=content_type,
-            json=_json,
-            content=_content,
-            headers=_headers,
-            params=_params,
-        )
-        request.url = self._client.format_url(request.url)  # type: ignore
-
-        pipeline_response = self._client._pipeline.run(  # type: ignore # pylint: disable=protected-access
-            request, stream=False, **kwargs
-        )
-
-        response = pipeline_response.http_response
-
-        if response.status_code not in [200]:
-            map_error(status_code=response.status_code, response=response, error_map=error_map)
-            raise HttpResponseError(response=response)
-
-        if response.content:
-            deserialized = response.json()
-        else:
-            deserialized = None
+    @abc.abstractmethod
+    def analyze_body(self, *args, **kwargs) -> str:
```

Comment:
Hi @iscai-msft,  I see currently all abstract methods are generated with parameters "*args, **kwargs", do we need to generate normal parameter signature for this case? 
~~~
@abc.abstractmethod
def analyze_body(self, input: Optional[Union[JSON, IO]] = None, **kwargs: Any) -> str:
~~~

Reply:
I think `*args, **kwargs` should still be good, are there issues you're running into?

Reply:
There will be mypy issues when changing the signature to below in the _patch.py, but should can be resolved by adding ignore:

~~~
def analyze_body(self, input: Optional[Union[JSON, IO]] = None, **kwargs: Any) -> str:     #typing: ignore
~~~